build.gradle

```diff
@@ -318,16 +344,38 @@ project(':core') {
     testCompile.exclude module: 'api-ldap-schema-data'
   }
 
-  tasks.create(name: "copyDependantLibs", type: Copy) {
-    from (configurations.testRuntime) {
-      include('slf4j-log4j12*')
-    }
+  tasks.create(name: "copyDependantLibsScala", type: Copy) {
     from (configurations.runtime) {
-      exclude('kafka-clients*')
+      include('*scala*.jar')
     }
-    into "$buildDir/dependant-libs-${versions.scala}"
+    into "$baseBuildDir/dependant-libs-scala-${versions.scala}"
   }
+  tasks.create(name: "copyTestDependantLibsScala", type: Copy) {
+    from (configurations.testRuntime - configurations.runtime) {
+      include('*scala*.jar')
+    }
+    into "$baseBuildDir/test-dependant-libs-scala-${versions.scala}"
+  }
+
+  task linkJars(dependsOn: ['copyDependantLibsJava', 'copyDependantLibsScala', 'copyTestDependantLibsJava', 'copyTestDependantLibsScala']) {
+    Files.createDirectories(Paths.get("$buildDir/dependant-libs-java"))
+    Files.createDirectories(Paths.get("$buildDir/dependant-libs-scala-${versions.scala}"))
+    Files.createDirectories(Paths.get("$buildDir/test-dependant-libs-java"))
+    Files.createDirectories(Paths.get("$buildDir/test-dependant-libs-scala-${versions.scala}"))
 
+    copyDependantLibsJava.eachFile {
```

Comment:
I don't think [eachFile](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.Copy.html#org.gradle.api.tasks.Copy:eachFile(groovy.lang.Closure)) is what you want since it executes the task before its copied.